tests/unit/esgf/test_search.py

```diff
@@ -119,7 +119,7 @@ def get_mock_connection(mocker, search_results):
     cfg = {
         'search_connection': {
             'urls': [
-                'https://esgf-index1.example.com/esg-search',
+                'https://esgf.example.com/esg-search',
                 'https://esgf-index2.example.com/esg-search',
```

Comment:
What I meant is this: why rename `index1` and leave `index2`? At least rename both if you want to get rid of the word `index`, e.g.
```suggestion
                'https://esgf-1.example.com/esg-search',
                'https://esgf-2.example.com/esg-search',
```
the numbering makes sense if you look at the tests in the file, so please keep it.